90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -977,10 +977,19 @@ populate_be_list() {
       # All other datasets are ignored
       continue
     fi
+    if [ -n "${BOOTFS}" ] && [ "${BOOTFS}" = "${fs}" ] ; then
```

Comment:
Do we need to test if `${BOOTFS}` is non-empty? The value of `${fs}` should never be empty so the test will still work as expected even when `${BOOTFS}` is empty.

Also, is there value in verifying that `${fs}` is nonempty at the start of the loop in Line 970?

Reply:
Agreed -  testing if BOOTFS is non-empty is largely pointless.  `${fs}` will always be defined when listing filesystems.